chacra/util.py

```diff
@@ -150,7 +150,7 @@ def get_extra_repos(project, ref=None, repo_config=None):
     return distinct_ref
 
 
-def get_extra_binaries(project_name, distro, distro_version, distro_versions=None, ref=None):
+def get_extra_binaries(project_name, distro, distro_version, distro_versions=None, ref=None, sha1=None):
```

Comment:
this makes `sha1` optional but it is always getting passed in. In what scenario it wouldn't be passed as an argument?